stack.yaml

```diff
@@ -1,4 +1,12 @@
-resolver: lts-9.17
+resolver: lts-11.9
 packages:
 - .
 - example-project
+extra-deps:
+  - amazonka-core-1.6.0
+  - amazonka-kinesis-1.6.0
+  - amazonka-s3-1.6.0
+nix:
+  packages:
+    - zlib.dev
```

Comment:
Just specifies that the program relies on the external (i.e. non-Haskell) `zlib` library (via [`zlib`](https://hackage.haskell.org/package/zlib)). This declaration should only come into effect if someone uses `stack`'s `nix` functionality and will otherwise be inert.